src/pudl/transform/ferc1.py

```diff
@@ -790,11 +789,27 @@ def check_table_calculations(
             f"{table_name}: has {len(off_df)} ({off_ratio:.02%}) records that don't "
             "calculate exactly."
         )
+
+    if off_ratio > 0 and off_ratio <= params.calculation_tolerance:
```

Comment:
I think we don't want the second half of this. a table can be off by less than the `calculation_tolerance` and i think we still want to add corrections to the off guys.

Reply:
or are you saying we don't add corrections if the table isn't so bad as to trip the assertion right under this section? okay maybe that is fine but it also seems a lil duplicative. if you take the second chunk away you could slip the corrections all under the `if off_ratio > 0:` above